examples/stripe-web-hook/index.ts

```diff
@@ -12,7 +12,7 @@ const corsHeaders = {
   "Access-Control-Allow-Headers": "authorization, x-client-info, apikey",
 };
 
-const stripe = Stripe(Deno.env.get("STRIPE_API_KEY"), {
+const stripe = Stripe("STRIPE_API_KEY", {
```

Comment:
I guess we would need to change this to get the environment variable?

Reply:
Right